wsdl/ver10/schema/onvif.xsd

```diff
@@ -7723,6 +7728,11 @@ and sample rate. </xs:documentation>
 					<xs:documentation>Token of a storage configuration.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="StorageStrategy" type="xs:string" minOccurs="0">
```

Comment:
is this parameter is applicable only for the camera which supports cloud object storage? I see StorageStrategy->Local also should support export recorded data to the external storage using ExportRecordedSegments.

Reply:
TargetConfiguration was added specifically for cloud recording, so no, it would not affect anything related to Profile G and local recording